tests/v0_tests/test_marqo_cloud_instance_mapping.py

```diff
@@ -134,6 +134,175 @@ def test_request_of_not_existing_index_raises_error(self, mock_get):
         with self.assertRaises(MarqoCloudIndexNotFoundError):
             mapping.get_index_base_url("index2")
 
+    @patch("requests.get")
+    def test_get_indexes_fails_cache_doesnt_update(self, mock_get):
```

Comment:
this is a good test. What about a similar test, but the first time trying to `get_indexes` fails?